tools/schema/cardBaseSchema.json

```diff
@@ -54,18 +54,15 @@
             "maxLength": 20,
             "pattern": "^[a-z]+_[0-9a-z]+$"
           },
+          "direction": {
+            "description": "Link direction; either towards this card, or outwards from this card.",
+            "type": "string",
+            "enum": ["outbound", "inbound"]
+          },
           "linkType": {
-            "description": "The link type is named according to the from-to direction. In user interfaces, the naming according to the link type is shown in the 'from' card.",
+            "description": "The link type is named according to the 'direction'. In user interfaces, the naming according to the link type is shown in the 'from' card.",
             "$comment": "The same link type names are used when indicating links in the index.json of 'to' cards, but the following reverse names shown in user interfaces: is blocked by, is caused by, has been instantiated to, is duplicate of, is mitigated by, is related to, is tested by",
```

Comment:
This $comment should be removed. Currently, we're recording the links to the "from" card only, and there is no hard-coded list of link types, so the list "is blocked by, is caused by" etc. is not helpful